UmbracoFood.Infrastructure/Repositories/OrderRepository.cs

```diff
@@ -152,5 +152,10 @@ public void SetOrderIsInDelivery(int orderId, DateTime estimatedDeliveryTime)
             var inDeliveryStatusId = (int)OrderStatus.InDelivery;
             _db.Execute("UPDATE Orders SET EstimatedDeliveryTime=@0, StatusId=@1 WHERE Orders.Id=@2", estimatedDeliveryTime, inDeliveryStatusId, orderId);
         }
+
+        public void RemoveOrderedMeal(int orderedMealId)
+        {
+            _db.Execute("DELETE FROM OrderedMeals Where Id = @0", orderedMealId);

```

Comment:
ad 2. Właśnie dlatego się zastanawiam czy nie powinniśmy zablokować usuwania nieaktualnych zamówień. Jest to swego rodzaju historia, by każdy mógł przypomnieć sobie co zamawiał i za ile